### PR TITLE
[EOS-24221] s3 setup remote invocation change

### PR DESCRIPTION
**Pull request by shalakadharap**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/ldap/setup_ldap.sh [hunk 1]
```diff
@@ -182,6 +180,10 @@ ldapmodify -D "cn=admin,cn=config" -w $ROOTDNPASSWORD -a -f $INSTALLDIR/ppolicyo
 
 ldapmodify -x -a -H ldapi:/// -D cn=admin,dc=seagate,dc=com -w "$ROOTDNPASSWORD" -f "$INSTALLDIR"/ppolicy-default.ldif || /bin/true
 
+# add S3 schema
+ldapadd -x -D "cn=admin,cn=config" -w $ROOTDNPASSWORD -f $INSTALLDIR/cn\=\{2\}s3user.ldif -H ldapi:///
```
Comment (bkirunge7) [inline scripts/ldap/setup_ldap.sh:184]: ldapi might work local only, can you check this

File: scripts/provisioning/s3setup_prereqs.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
   "init": {
     "rpms": ["symas-openldap", "symas-openldap-servers", "symas-openldap-clients", "haproxy", "cortx-motr", "python36-ldap"],
     "pip3s": ["s3confstore", "cryptography", "s3cipher"],
-    "services": ["sshd", "haproxy", "slapd", "rsyslog"],
+    "services": ["sshd", "haproxy", "slapd", "rsyslogd"],
```
Comment (bkirunge7) [inline scripts/provisioning/s3setup_prereqs.json:40]: pre-requisite for slapd might fail for remote ldap right?

Comment (shalakadharap) [reply scripts/provisioning/s3setup_prereqs.json:40]: validation will not run for remote node

File: scripts/ldap/setup_ldap.sh (hunk 1 above)
Comment (shalakadharap) [reply scripts/ldap/setup_ldap.sh]: This is not the file which is getting used.

**Pull request merged**